src/main/java/uk/gov/hmcts/reform/coh/controller/utils/CohUriBuilder.java

```diff
@@ -4,27 +4,46 @@
 
 public class CohUriBuilder {
 
+    public static String buildOnlineHearingPost() {
+
+        return "/continuous-online-hearings";
+    }
+
     public static String buildOnlineHearingGet(UUID onlineHearingId) {
 
-        return String.format("/continuous-online-hearings/%s", onlineHearingId);
+        return String.format(buildOnlineHearingPost() + "/%s", onlineHearingId);
+    }
+
+    public static String buildQuestionPost(UUID onlineHearingId) {
+
+        return buildOnlineHearingGet(onlineHearingId) + "/questions";
     }
 
     public static String buildQuestionGet(UUID onlineHearingId, UUID questionId) {
 
-        return String.format("/continuous-online-hearings/%s/questions/%s", onlineHearingId, questionId);
+        return String.format(buildQuestionPost(onlineHearingId) + "/%s", questionId);
```

Comment:
Switching between String.format & concatenation in this class. Let's just use one?